tests/unit/stack/test_stack_component_class_registry.py

```diff
@@ -136,19 +123,19 @@ def _activate_integrations():
     # getting the local orchestrator should not require activating integrations
     StackComponentClassRegistry.get_class(
         component_type=StackComponentType.ORCHESTRATOR,
-        component_flavor=OrchestratorFlavor.LOCAL,
+        component_flavor="local",
```

Comment:
I'm wondering if we should at least have string constants somewhere for all the ZenML internal flavors, what do you think?